sonar-dotnet-shared-library/src/main/java/org/sonarsource/dotnet/shared/plugins/FileTypeSensor.java

```diff
@@ -32,7 +32,7 @@
 
 /**
  * This class is a non-global sensor used to count the type of files in the .NET projects (i.e. Scanner modules).
- *
+ * <p>
```

Comment:
this is added automatically by IntelliJ, I think it's a javadoc standard

Reply:
It's for paragraphs separation. See: https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html